doc/source/user_guide/troubleshoot.rst

```diff
@@ -594,36 +596,40 @@ your program or script so that you can turn on and off logging and
 verbosity as needed.
 
 
-Issues
-~~~~~~
-
-.. note::
+Known Issues
+~~~~~~~~~~~~
 
-   MAPDL 2021 R1 has a stability issue with the :
-   :func:`Mapdl.input() <ansys.mapdl.core.Mapdl.input>`
-   method. Avoid using input files if possible. Attempt to use the
-   :func:`Mapdl.upload() <ansys.mapdl.core.Mapdl.upload>` method to upload
-   nodes and elements and read them in via the
-   :func:`Mapdl.nread() <ansys.mapdl.core.Mapdl.nread>` and
-   :func:`Mapdl.eread() <ansys.mapdl.core.Mapdl.eread>` methods.
+* MAPDL 2021 R1 has a stability issue with the :
+  :func:`Mapdl.input() <ansys.mapdl.core.Mapdl.input>`
+  method. Avoid using input files if possible. Attempt to use the
+  :func:`Mapdl.upload() <ansys.mapdl.core.Mapdl.upload>` method to upload
+  nodes and elements and read them in via the
+  :func:`Mapdl.nread() <ansys.mapdl.core.Mapdl.nread>` and
+  :func:`Mapdl.eread() <ansys.mapdl.core.Mapdl.eread>` methods.
 
 
 
 More help needed?
 -----------------
 
+.. vale off
+
 .. epigraph::
 
-   *"What do you do if a problem is not listed here?"*  
+   *"What do I do if a problem is not listed here?"*  
```

Comment:
Earlier you used the term "issue" and here "problem". I'd use "issue" to be consistent. Always using the same terminology helps users to understand the content better.

Reply:
Good call!